ja-jp/index.html

```diff
@@ -60,7 +59,7 @@
                 </a>
               </div>
               <div class="col-2 align-self-center">
-                <a href="../index.html" class="text-white">
+                <a href="/" class="text-white">
```

Comment:
Please use relative paths, makes it easier to develop/debug.

Reply:
Already fixed in my repo :)
https://github.com/pklion/files-community.github.io/blob/translation/ja-jp/index.html

Reply:
```suggestion
                <a href="../" class="text-white">
```